requirements.txt

```diff
@@ -1,5 +1,8 @@
-# workaround for https://github.com/celery/celery/issues/3225
-amqp>1.4,<2.0
+# recommended broker is py-amqp(amqp) for celery (librabbitmq is python2 only):
+# https://github.com/celery/librabbitmq/issues/118#issuecomment-385461136
+# https://github.com/celery/librabbitmq/issues/119#issuecomment-388327748
+# http://docs.celeryproject.org/en/3.1/userguide/optimizing.html?highlight=pyamqp#librabbitmq-python-2-only
+amqp==1.4.9
```

Comment:
This is not needed: `amqp` is installed by `kombu` which is installed by `celery`. Listing `amqp` explicitly was only necessary as a workaround for a bug in `librabbitmq`.

Reply:
that makes sense - i will remove then